### PR TITLE
Adding alternate codelabs link.

### PR DESCRIPTION
**Pull request by Sfshaza**
Please review @kwalrath.

Staged:  https://sz-flutter-2.firebaseapp.com/codelabs/

### CODE REVIEW

File: codelabs/index.md [hunk 1]
```diff
@@ -25,9 +25,13 @@ A deeper "first dive" than "Write Your First Flutter App." In this codelab
 you'll create a chat app that includes a simple animation, and customizes
 the UI for iOS and Android.
 
+---
+
 For a full list of available Flutter codelabs, see the
 [Flutter category](https://codelabs.developers.google.com/?cat=Flutter)
 on [Google Developers](https://codelabs.developers.google.com/).
+If you are in a country where the previous link doesn't work, try
+[Flutter Codelabs](https://codelabs.flutter-io.cn/).
```
Comment (kwalrath) [inline codelabs/index.md]: The link text could be a little clearer. Maybe:

[this mirror of the Flutter codelabs]

File: codelabs/index.md [hunk 2]
```diff
@@ -25,9 +25,13 @@ A deeper "first dive" than "Write Your First Flutter App." In this codelab
 you'll create a chat app that includes a simple animation, and customizes
 the UI for iOS and Android.
 
+---
+
 For a full list of available Flutter codelabs, see the
 [Flutter category](https://codelabs.developers.google.com/?cat=Flutter)
 on [Google Developers](https://codelabs.developers.google.com/).
+If you are in a country where the previous link doesn't work, try
```
Comment (kwalrath) [inline codelabs/index.md]: "in a country" and "previous link" seem possibly confusing. Maybe just:

If the previous links don't work for you, try

File: codelabs/index.md [hunk 3]
```diff
@@ -25,9 +25,13 @@ A deeper "first dive" than "Write Your First Flutter App." In this codelab
 you'll create a chat app that includes a simple animation, and customizes
 the UI for iOS and Android.
 
+---
+
 For a full list of available Flutter codelabs, see the
 [Flutter category](https://codelabs.developers.google.com/?cat=Flutter)
 on [Google Developers](https://codelabs.developers.google.com/).
+If the previous link doesn't work for you, try
+[this mirror of the Flutter Codelabs](https://codelabs.flutter-io.cn/).
```
Comment (kwalrath) [inline codelabs/index.md]: Codelabs -> codelabs

**Comment by Sfshaza** [comment]
I'll catch that C->c on the next one.  ;)

**Pull request merged**